### PR TITLE
Added PCM Linear interpolation.

### PR DESCRIPTION
**Pull request by bcc2528**

### CODE REVIEW

File: src/rf5c68/rf5c68.cpp [hunk 1]
```diff
@@ -47,6 +47,20 @@ static inline int Gain(int a,int b)
 	}
 }
 
+// Linear interpolation
+static inline int lerp_int(int start, int end, float t)
+{
+	if (t <= 1.0)
```
Comment (fuzz6001) [inline src/rf5c68/rf5c68.cpp:53]:
```suggestion
	if (t >= 1.0)
```

Comment (bcc2528) [reply src/rf5c68/rf5c68.cpp:53]: thank you.

**Comment by bcc2528** [comment]
waveform

before
before


after (linear interpolation)
after

**Comment by captainys** [comment]
Thanks!

**Pull request merged**